### DISCUSSION TITLE
Reset Zoom

### DISCUSSION THREAD

**Discussion by neuralgitwork** [freeplane/freeplane#926] [Help] [answered]
i cant figure out how to reset the zoom level.

once i changed it to a bigger zoom level and closed it (and reopened it of course) i forgot what the "intended" zoom level was

(sry if this is a dumbo question, then ill delete this discussion again)

**Comment by neuralgitwork** [comment]
i figured i could just open the mindmap.mm file with a text editor and change the zoom attribute from zoom="1.3" back to 1.0 or even just delete it...

thats nice! 

am i really to blind to find this setting in the program's UI ??? i guess so XD

**Comment by abc16361** [comment]
Zoom in and Zoom out can be found on the menu View -> Zoom, and on the Tool bar left, next to the map save button, there is a dropdown and zoom levels can be selected. I can't find the button or menu to reset map zoom to 100%, maybe they are hidden somewhere, so I get a script and bind it with a shortcut to make it work. Hopefully this information helps.

Code:
`c.setZoom(1.0)`

**Reply by neuralgitwork** [reply]
thats a nice way to solve it,
(ill wait until i mark this as answer, maybe there is a button somewhere in the ui that we missed)

**Comment by dpolivaev** [comment, marked as answer]
I don't understand why you have this problem. There is  a zoom drop-down box in the tool bar.
<img width="72" alt="zoom" src="https://user-images.githubusercontent.com/1558052/211197953-f57649f7-c3cf-4f93-9548-10c4f17c8483.png">

**Reply by neuralgitwork** [reply]
oh ok i see ... i didnt have toolbar enabled to see more on my tiny laptop screen

... so its nowhere in the menu but its in the toolbar

thanks!!
